src/components/input/Input.vue

```diff
@@ -218,6 +240,47 @@ export default {
             this.$nextTick(() => {
                 this.$refs.input.focus()
             })
+        },
+		
+		shouldUpdateLocalValue (localValue, checkVal, fromBlur = false) {
+            return this.isNotInFocusAndDirty(localValue, checkVal, fromBlur)||this.isDirtyWithModifiers(localValue, checkVal, fromBlur);
+        },
+		
+        isNotInFocusAndDirty (localValue, checkVal, fromBlur) {
+            if(!fromBlur){//from value (v-model) watcher
+                if(this.isFocused && !this.forcedSync){
```

Comment:
If its focused and not forcedSync, for example on typing, then check if there trim or number stuff and then ignore local updates, same as v-model.trim/.number, they are updates onBlur event, but what about situations where we need instan updates, so, v-model with computed also support it, that's why it's here.